envoy/api/v2/core/BUILD

```diff
@@ -79,6 +79,19 @@ api_go_proto_library(
     ],
 )
 
+api_proto_library(
+    name = "envoy_http_target",
```

Comment:
I think "envoy" is a bit redundant here. And what about just calling this "http_uri" ?

Reply:
Done